### ISSUE TITLE
Code coverage reports in pull requests

### ISSUE THREAD

**Issue by seluianova** [digitalfabrik/entitlementcard#1682]
**Is your feature request related to a problem? Please describe.**
Currently we have code coverage reports generation in the administration and backend modules.
It would be nice to post some summary of these reports in the comments of pull requests.

**Describe the solution you'd like**
To be discussed, but I think would be nice to have following:
- % of total coverage and diff
- % of coverage of the changed code
- link to the detailed html report
- publish only reports related to the changed module. For example, no need to publish a backend report if only administration has been changed.

**Describe alternatives you've considered**
TBD

**Additional context**
Probably https://github.com/digitalfabrik/entitlementcard/issues/1356 should be done first

**Comment by seluianova** [comment]
Not completed, only the administration part was done

**Comment by connium** [comment]
Will be addressed in #2548
